spec/type_aliases/plugin_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'spec_helper'
+
+describe 'Letsencrypt::Plugin' do
+  it { is_expected.to allow_values('apache', 'standalone', 'webroot', 'nginx', 'dns-route53', 'dns-google') }
```

Comment:
please add this:
```
it { is_expected.to compile.with_all_deps }
```

that is the minimal test that should be present in all spec files. It ensures that the catalog compiles.

Reply:
This isn't working for data types:

```
     NameError:
       undefined local variable or method `compile' for #<RSpec::ExampleGroups::LetsencryptPlugin:0x00000000055c3118>
```

Reply:
Yes, type aliases don't really compile. Testing whether it allows and disallows certain values is sufficient testing though.